.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Debug Flask app",
+            "type": "python",
+            "request": "launch",
+            "python": "${fileWorkspaceFolder}/backend/image-upload-api/venv/Scripts/python.exe",
+            "program": "backend/image-upload-api/flask-api/main.py",
+            "args": [],
+            "jinja": true
+        }
+    ]
+}
```

Comment:
@YiminArava4508 @harperklauke -- This is a configuration file for VS Code. It tells VS Code how to start the Flask app so that it can attach a debugger to it. Remind me in a future meeting to show you guys how to use the debugger, it can be super helpful